### PR TITLE
Feature/credentials consolidation fixes AB#137

### PR DESCRIPTION
**Pull request by KimForss**
## Problem
Provide a consolidated solution for credentials management for both username/password and sshkeys

## Solution
<Please elaborate the solution for the problem>

## Tests
<Please provide steps to test the PR>

## Notes
<Additional comments for the PR>

### CODE REVIEW

File: deploy/terraform/terraform-units/modules/sap_system/common_infrastructure/variables_local.tf [hunk 1]
```diff
@@ -154,7 +148,7 @@ locals {
   anchor_authentication       = try(local.anchor.authentication, local.db_auth)
   anchor_auth_type            = try(local.anchor.authentication.type, "key")
   enable_anchor_auth_password = local.deploy_anchor && local.anchor_auth_type == "password"
-  enable_anchor_auth_key      = !local.enable_anchor_auth_password
+  enable_anchor_auth_key      = ! local.enable_anchor_auth_password
```
Comment (hdamecharla) [inline deploy/terraform/terraform-units/modules/sap_system/common_infrastructure/variables_local.tf:151]: Hi @KimForss, Should this be !local.enable_anchor_auth_password (as it was origianlly); I am not sure HCL would like a space between ! and the ```<expression>```

Comment (KimForss) [reply deploy/terraform/terraform-units/modules/sap_system/common_infrastructure/variables_local.tf:151]: I think running terraform fmt adds the space - will check

Comment (KimForss) [reply deploy/terraform/terraform-units/modules/sap_system/common_infrastructure/variables_local.tf:151]: Yes- it is terraform fmt that adds the space

**Review by hdamecharla — APPROVED** [review]
Hi @KimForss, Approved #1066

**Pull request merged**